ngrinder-runtime/src/main/java/org/ngrinder/utils/CommonUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.ngrinder.utils;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang.StringUtils;
+
+import java.io.File;
+
+public class CommonUtils {
+	public static String resolveHome(String homeEnv, String homePropery, String homeFolderName) {
+		String userHome = StringUtils.defaultIfEmpty(homePropery, homeEnv);
+
+		if (StringUtils.isEmpty(userHome)) {
+			userHome = System.getProperty("user.home") + File.separator + homeFolderName;
+		} else if (StringUtils.startsWith(userHome, "~" + File.separator)) {
+			userHome = System.getProperty("user.home") + File.separator + userHome.substring(2);
+		} else if (StringUtils.startsWith(userHome, "." + File.separator)) {
+			userHome = System.getProperty("user.dir") + File.separator + userHome.substring(2);
+		}
+		return FilenameUtils.normalize(userHome);
+	}
```

Comment:
Runtime is not for common utils. Please relocate this in the other place.

Reply:
yes, I will remove this. then I apply this for each modules.

How about making ngrinder-util modules for the entire project? Is it unnecessary?

Reply:
We can make it when there is more needs for it.